### PR TITLE
Quiz-Card

### PR DESCRIPTION
**Pull request by tillwilling**
Work in progress

### CODE REVIEW

**Comment by pascalweber85** [comment]
hübsch

File: src/styles/quiz-card/quiz-card.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: green;
+}
```
Comment (macfrei) [inline src/styles/quiz-card/quiz-card.css:3]: This should be in the `_base.css` file :)

File: src/styles/quiz-card/quiz-card.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<section class="quiz-card__card">
```
Comment (macfrei) [inline src/styles/quiz-card/quiz-card.html:1]: This would just be:
```suggestion
<section class="quiz-card">
```

**Review by macfrei — COMMENTED** [review]
Looks good for now, please note that generic styles like for the `body` should be in another file! Keep up the good work!

File: src/styles/_utility.css [hunk 3]
```diff
@@ -5,3 +5,14 @@
 .m0 {
   margin: 0 !important;
 }
+
+.df > li {
```
Comment (Morten-Moeller) [inline src/styles/_utility.css:9]: Its a bit complex for a utility class. Its probably better to add this styles in the component.

Comment (Morten-Moeller) [reply src/styles/_utility.css:9]: add this to your quiz-card.css

Comment (tillwilling) [reply src/styles/_utility.css:9]: done, thank you Morten! :)

**Review by Morten-Moeller — COMMENTED** [review]
Very good

**Pull request merged**